shell/platform/darwin/macos/framework/Source/FlutterGLCompositor.h

```diff
@@ -0,0 +1,79 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <map>
+
+#include "flutter/fml/macros.h"
+#include "flutter/shell/platform/darwin/macos/framework/Source/FlutterSurfaceManager.h"
+#include "flutter/shell/platform/darwin/macos/framework/Source/FlutterViewController_Internal.h"
+#include "flutter/shell/platform/embedder/embedder.h"
+#include "third_party/skia/include/gpu/GrDirectContext.h"
+
+namespace flutter {
+
+// FlutterGLCompositor creates and manages the backing stores used for
+// rendering Flutter content and presents Flutter content and Platform views.
+// Platform views are not yet supported.
+// FlutterGLCompositor is created and destroyed by FlutterEngine.
+class FlutterGLCompositor {
+ public:
+  FlutterGLCompositor(FlutterViewController* view_controller);
+
+  // Creates a BackingStore and saves updates the backing_store_out
+  // data with the new BackingStore data.
+  // If the view requesting the backing store is the root view,
```

Comment:
What is root view? Is it the flutter view? Maybe call it `FlutterView` instead of root view?